webpack.config.dev.js

```diff
@@ -12,15 +12,12 @@ const insertHotReload = (host, port, entries) => (
   R.map(R.compose(R.flatten, v => [v].concat(hotEntry(host, port))), entries)
 );
 
-const devConfig = {
+const devConfig = Object.assign(config, {
```

Comment:
I'm not following what you said about `devConfig.module.loaders` below, do you mean that the `devConfig` loaders won't be affected? They are, otherwise the app wouldn't boot (because `babel-loader` wouldn't have been added).